spec/lib/task-graph-spec.js

```diff
@@ -520,6 +520,14 @@ describe('Task Graph', function () {
                     .that.equals('finished');
             });
         });
+
+        it('should create task with waitOn.either', function(){
```

Comment:
@zhangy42 : If we can do it in unit-test, that will be best. Otherwise, do in in FIT is also acceptable.